contracts/MechFixedPriceToken.sol

```diff
@@ -5,13 +5,16 @@ import {OlasMech} from "./OlasMech.sol";
 
 /// @title MechFixedPriceToken - Smart contract for OlasMech that accepts a fixed price payment for services in native token.
 contract MechFixedPriceToken is OlasMech {
+    // keccak256(FixedPriceToken) = 3679d66ef546e66ce9057c4a052f317b135bc8e8c509638f7966edfd4fcf45e9
+    bytes32 public constant PAYMENT_TYPE = 0x3679d66ef546e66ce9057c4a052f317b135bc8e8c509638f7966edfd4fcf45e9;
```

Comment:
Hashing approach always wins :). Although in this case it feels like we maybe need to call it FixedPriceOlas? Either way is fine.

Reply:
hm I think the token can be different kinds for the same factory contract (different factory instance) no?

Reply:
Makes sense, maybe encode it on a level of factory? There we can combine `FixedPriceToken` + token address or anything else.

Reply:
Hm, I'll think about it.